sdk/core/azure-core-management/pom.xml

```diff
@@ -59,7 +59,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.9.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core;current} -->
+      <version>1.8.1</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
```

Comment:
Can you also confirm that README and CHANGELOG are updated?

Reply:
CHANGELOG and README were updated in this PR https://github.com/Azure/azure-sdk-for-java/pull/15480